src/index.tsx

```diff
@@ -32,6 +29,39 @@ if (browser.name === 'ie') {
   );
 }
 
+axios.interceptors.request.use(
```

Comment:
@kahlouie @Igarfinkle We no longer need to pass access tokens to requests made to `process.env.REACT_APP_API_ADDRESS`. This [axios interceptor](https://github.com/axios/axios#interceptors) "intercepts" the request and adds the authorization header. Axios ONLY intercepts requests made to our API address.

Reply:
this is very nifty, thank you